docs/i2c_driver.md

```diff
@@ -190,7 +190,7 @@ Receive multiple bytes from the selected SPI device.
 
 ### `i2c_status_t i2c_writeReg(uint8_t devaddr, uint8_t regaddr, uint8_t* data, uint16_t length, uint16_t timeout)`
 
-Writes to a register on the I2C device.
+Writes to an 8-bit register on the I2C device.
```

Comment:
Sorry, I do actually have an issue with a wording here. When I an most people I know say X-bit register we usually refer to the size of the data portion of the register. But what's 8 bits here is the address of the register, not its data. It should be something like "writes to a register with a 8-bit address".

Reply:
I kinda think the datatype of `regaddr` would give you a clue here, and in all cases you give the function a pointer to the data, and the number of bytes to read, rather than a single 8/16-bit value.